src/core/timeout.ts

```diff
@@ -0,0 +1,10 @@
+import { Event } from "@clarity-types/data";
+import wrap from "./wrap";
+
+export function setTimeout(handler: (event?: Event | boolean) => void, timeout: number, event?: Event): number {
+    return window.setTimeout(wrap(handler), timeout, event);
```

Comment:
>setTimeout [](start = 18, length = 10)

Use a requestIdleCallback if available?

Reply:
Not adding extra complexity since not all modern browsers support requestIdleCallback. It's still an experimental technology and pollyfills rely on setTimeout / requestAnimationFrame which we are using. We can revisit using requestIdleCallback when it's more mainstream. Also, I am not sure if that's what we really want based on the descript. We can chat more offline and talk about it future versions.